pallets/moonbeam-xcm-benchmarks/src/mock.rs

```diff
@@ -51,7 +51,9 @@ impl xcm_executor::traits::OnResponse for DevNull {
 	}
 }
 
+#[allow(dead_code)]
 pub struct AccountIdConverter;
+
```

Comment:
Removing `pub` from `pub struct AccountIdConverter;` should be enough.

Reply:
Just gave it a try, but unfortunately it is not enough.